cmd/validator/validator.go

```diff
@@ -91,6 +94,19 @@ func validateFileTypeList(input []string) bool {
 	return true
 }
 
+func buildSearchPath(n int) []string {
+	searchPaths := make([]string, 0)
+	// If search path arg is empty, set it to the cwd
+	// if not, set it to the arg. Supports n number of
+	// paths
+	if n == 0 {
+		searchPaths = append(searchPaths, ".")
+	} else {
+		searchPaths = append(searchPaths, flag.Args()...)
+	}
+	return searchPaths
+}
```

Comment:
This sounds a bit complicated to me.

The code was copy pasted for previous implementation that somehow needed it.

It can be simpler now.

What about this?


```suggestion
func buildSearchPath(n int) []string {
	// If search path arg is empty, set it to the cwd
	if n == 0 {
		return []string{"."}
	}

	// if not, set it to the arg. Supports n number of
	// paths
	return flag.Args()
}
```

Reply:
See also 

https://github.com/Boeing/config-file-validator/pull/192#discussion_r1819473081

As it might change the way you handle things